web/typescript/main.ts

```diff
@@ -43,6 +44,9 @@ function pageSpecificFunctions(settings: UserSettings) {
             saveRedirect();
             break;
         case '/account':
+            checkMaxLength();
+            checkRedirect(settings.redirect);
```

Comment:
Should we be calling checkRedirect here?

Reply:
I don't know but we were [originally](https://github.com/userstyles-world/userstyles.world/blob/dd1aebdf8bf19ddb603ed7b7d1ad625279d27a3a/web/typescript/main.ts).

Reply:
Ah, my bad. I thought this was not the case originally.